13-RemoteRepositorySetup/README.md

```diff
@@ -0,0 +1,75 @@
+# Working with remote repository
+
+## Introduction
+
+* So far we worked on local repository.
+* In real world you would have a remote copy for several reasons:
+  * Backup of the code (remote repository is backup on its own but the server
+    usually has a backup too).
+  * You would have a central repository shared by multiple developers.
+* Since _GIT_ is distributed source control management system you
+  usually have multiple copies of the repository on various places.
+  * All the copies are exactly same and equivalent.
+  * You can synchronize with any repository around.
+* Central repository usually gives you advices on how to publish
+  the repository for the first time.
+
+## Commands
+
+* Let's publish our previously created repository:
+  * Create an empty repository on your favorite server (for example _Github_).
+* Setup the remote repository for the local repository:
+```
+> git remote add origin https://github.com/voloda/GitTrainingPlayground.git
```

Comment:
maybe add some note that they should replace your username with their own? possible ...github.com/$USERNAME/$REPONAME... or something?